packages/theme-chalk/src/menu.scss

```diff
@@ -172,6 +172,7 @@
         2
     );
 
+    > .#{$namespace}-menu-item-group .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title,
     > .#{$namespace}-menu-item,
     > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title {
```

Comment:
```suggestion
    > .#{$namespace}-menu-item,
    > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title,
    > .#{$namespace}-menu-item-group > ul > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title {
```

You should avoid affecting other styles

![2022-12-01 17.26.58.png](https://img1.imgtp.com/2022/12/01/lhkvMLMX.png)

Reply:
One more tag ul?
ok, i got it, so good~